.github/workflows/setup.yml

```diff
@@ -0,0 +1,46 @@
+name: Distribution
+
+on: [push, pull_request]
+
+jobs:
+  setup:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Checkout submodules
+      shell: bash
+      run: |
+        auth_header="$(git config --local --get http.https://github.com/.extraheader)"
+        git submodule sync --recursive
+        git -c "http.extraheader=$auth_header" -c protocol.version=2 submodule update --init --force --recursive --depth=1
+        git config --global user.email "ci@example.com"
+        git config --global user.name "CI Job"
+    - name: Cache pip
+      uses: actions/cache@v1
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-setup-${{ hashFiles('**/requirements*.txt') }}
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install apt dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y libgettextpo-dev
+    - name: Install pip dependencies
+      run: |
+        pip install -r requirements/dev.txt
+        pip install twine virtualenv
+    - name: Build sdist
+      run: make build
+    - name: Twine check
+      run: twine check dist/*
+    - name: Install test
+      run: |
+        virtualenv test-ttk-release
```

Comment:
Will this use system Python (2.7 I guess) or the 3.7 runtime that was just installed? You may need to use `vitualenv -p python3.7`

Reply:
This will use Python defined above in the workflow, so 3.7.